src/js/elements/mapchip/mapchip.js

```diff
@@ -127,6 +127,9 @@ export class MapChip extends Component {
         // Tooltip
         this._tooltip.enableTooltip(this._toggleIconDownContainer, "Collapse Details");
         this._tooltip.enableTooltip(this._toggleIconUpContainer.parent(), "Expand Details");
+
+        // Snackbar
+        $("<div id='mapchip-snackbar'></div>").insertBefore(".map-bottom-items--v2 .map-point-legend");
```

Comment:
can you replace this with
```
 $("<div id='mapchip-snackbar'></div>").insertBefore(`${mapChipBlockClass} .map-point-legend`);
```

so in future when we want to rename the `map-bottom-items` to v3 or something it would be easy